.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "snap" ]
```

Comment:
Why does snap need to live on it's own branch? Can't we merge it into main?

Reply:
I did that for testing... I forgot to remove that. :smiling_face_with_tear:

Reply:
Really got frustrated while fixing this symbolic icon thing.